flink-examples/flink-examples-streaming/src/test/scala/org/apache/flink/streaming/scala/examples/StreamingExamplesITCase.scala

```diff
@@ -23,8 +23,7 @@ import java.io.File
 import org.apache.commons.io.FileUtils
 import org.apache.flink.core.fs.FileSystem.WriteMode
 import org.apache.flink.streaming.api.TimeCharacteristic
-import org.apache.flink.streaming.api.scala._
-import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
+import org.apache.flink.streaming.api.scala.{StreamExecutionEnvironment, _}
```

Comment:
nit: The import looks strange. I think

```
import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
```
is enough.

Reply:
Will change it.